mas-foundation/build.gradle

```diff
@@ -137,60 +135,9 @@ task javadoc(type: Javadoc) {
     failOnError false
 }
 
-//jcenter integration
 group 'com.ca'
 
 project.version = mas_version
-bintray {
```

Comment:
I assume, we should do the same for other modules in this repo. Don't we?

Reply:
yes.. we are going to do the same for other repos.